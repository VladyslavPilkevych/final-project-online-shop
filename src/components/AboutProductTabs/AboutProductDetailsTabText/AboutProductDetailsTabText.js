import React from 'react';

function AboutProductDetailsTabText() {
  return (
    <div>
      AboutProductDetailsTabText
    </div>
  );
}

export default AboutProductDetailsTabText;
