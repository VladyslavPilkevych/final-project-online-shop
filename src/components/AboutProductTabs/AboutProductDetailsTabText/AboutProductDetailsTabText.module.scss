@import '../../../assets/styles/global.module.scss';

.productTitle {
    margin-top: 83px;
    color: $black;
    font-family: $fonts;
    font-style: normal;
    font-weight: 500;
    font-size: 32px;
    line-height: 48px;
}

ul.listWrapper{
    list-style-type: disc;
    display: block !important;
    margin-top: 55px;
    
    & li {
        color: $black;
        font-family: $fonts;
        font-style: normal;
        font-weight: 300;
        font-size: 14px;
        line-height: 24px;
    }
}

.itemNo {
    text-align: right;
    margin-top: 20px;
}

.contactUsText {
    font-family: $fonts;
    font-style: Bold;
    font-weight: 600;
    font-size: 12px;
    line-height: 18px;
    padding-bottom: 100px;
    margin-top: 20px;
}