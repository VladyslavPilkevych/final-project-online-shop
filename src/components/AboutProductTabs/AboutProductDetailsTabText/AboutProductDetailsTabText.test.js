import { render } from '@testing-library/react';
import AboutProductDetailsTabText from './AboutProductDetailsTabText';

describe('Snapshot AboutProductDetailsTabText should render', () => {
    test('Snapshot AboutProductDetailsTabText is rendered', () => {
      const { asFragment } = render(<AboutProductDetailsTabText />);
      expect(asFragment()).toMatchSnapshot();
    });
  });