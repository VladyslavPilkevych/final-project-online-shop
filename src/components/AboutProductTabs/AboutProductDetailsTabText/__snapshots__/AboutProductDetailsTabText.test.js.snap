// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot AboutProductDetailsTabText should render Snapshot AboutProductDetailsTabText is rendered 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="productTitle"
    >
       
    </h1>
    <ul
      class="listWrapper"
    />
    <p
      class="itemNo"
    >
      Article: 
    </p>
    <p
      class="contactUsText"
    >
      Have a Question? 
      <a
        href="/contact-us"
      >
        Contact us
      </a>
    </p>
  </div>
</DocumentFragment>
`;
